pt-BR/mail/chrome/messenger/messengercompose/messengercompose.dtd

```diff
@@ -320,129 +320,129 @@
 <!ENTITY formatToolbarPaddingLeftMac "75">
 
 <!-- topic context menu -->
-<!ENTITY pbTopicContextMenu-edit.label         "Edit">
+<!ENTITY pbTopicContextMenu-edit.label         "Editar">
 <!ENTITY pbTopicContextMenu-edit.accesskey     "E">
-<!ENTITY pbTopicContextMenu-favorite.label     "Favorite">
+<!ENTITY pbTopicContextMenu-favorite.label     "Favorito">
 <!ENTITY pbTopicContextMenu-favorite.accesskey "F">
-<!ENTITY pbTopicContextMenu-remove.label       "Remove">
+<!ENTITY pbTopicContextMenu-remove.label       "Remover">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY pbTopicContextMenu-remove.label       "Excluir">